.github/workflows/release-opam.yml

```diff
@@ -0,0 +1,38 @@
+name: opam release
+
+on:
+  release:
+    types: [published]
```

Comment:
So as far I understand this will be executed once release is created? Will it automatically bump the correct version?

Reply:
Yes, as soon as a new release is created (for example 2.1.1) this job runs and creates a pull request with the newest tag to the opam repository.